Make compatible with Panacea

youngjoon-lee opened the pull request:
This git repository is forked from https://github.com/LedgerHQ/app-cosmos which was published as the official Cosmos Ledger S/X App v2.18.0 (in Ledger Live).
![image](https://user-images.githubusercontent.com/5462944/109638481-6a5ebc80-7b91-11eb-90fe-3d6a11e1894c.png)

I made it work with Panacea by changing some parameters.
I tested it on a real Ledger S device. Unfortunately, I wasn't able to test it with Ledger X because the `make load` doesn't work with Ledger X.

TODO: make `make cpp_test` work.

cl9200 reviewed (approved):
:+1:

gyuguen commented on docs/TXSPEC.md:
It seems that our chain has set a fee of 5umed and 200000 gas.

gyuguen commented on tests/json_parser.cpp:
It seems that you need to change the atom to med here.

youngjoon-lee commented on docs/TXSPEC.md:
This document is only for an example. The setting of 5umed and 200000gas is configured in each network. So, this ledger implementation doesn't need to care about those settings.

youngjoon-lee commented on tests/json_parser.cpp:
Actually, it doesn't matter when considering the way how this test works, but I will change it for clarity.

gyuguen commented on tests/testcases/manual.json:
So is this
```suggestion
            "amount": "5",
```

youngjoon-lee commented on docs/TXSPEC.md:
Specifically, the ledger app always uses configurations that are specified by `--fees`, `--gas` and `--gas-price`.

youngjoon-lee commented on tests/testcases/manual.json:
We don't need to change this value.
https://github.com/medibloc/ledger-panacea/pull/1#discussion_r853955886

youngjoon-lee commented on tests/testcases/manual.json:
Actually, we also don't need to change the chain ID from cosmoshub-4 to panacea-3.

gyuguen commented on tests/tx_parse.cpp:
It is also marked as an atom here, is there any problem?

gyuguen commented on docs/TXSPEC.md:
I got it. Thank you

gyuguen reviewed (approved):
Thank you for your hard work.

youngjoon-lee commented:
@gyuguen @hansol-medi 
Changed the app name from `Panacea` to `MediBloc`: [91bf466](https://github.com/medibloc/ledger-panacea/pull/1/commits/91bf46612a021d10ab7f1eaf70bad2425c91e988)
Changed app icons: [71db396](https://github.com/medibloc/ledger-panacea/pull/1/commits/71db3962df0fceaa9557aaa1decde759012ba256)

0xHansLee reviewed (approved):
Thank you for your efforts!! 
We have our ledger app 🙌
